components/layouts/main/index.tsx

```diff
@@ -1,12 +1,20 @@
 import Footer from "./footer";
 import Header from "./header";
 
+const updateDate = '2022. 10. 02.';
+const versionNumber = '01';
+
 export default function MainLayout(props: { children: React.ReactNode }) {
   return (
-    <div className="w-[62rem] mx-auto">
-      <Header />
-      <main className="pb-16">{props.children}</main>
-      <Footer />
+    <div className="min-h-full relative">
+      <div className="w-[62rem] mx-auto min-h-screen">
+        <Header />
+        <main className="pb-16">{props.children}</main>
+      </div>
+      <Footer 
```

Comment:
Let's move the `Footer` after `main` to have matching `header`, `main`, and `footer` tags.